### PR TITLE
fix: add ImportError field to ImportStatus

### PR DESCRIPTION
**Pull request by zbindenren**

### CODE REVIEW

**Comment by zbindenren** [comment]
Otherwise it is impossible to find out import error.

File: project_import_export.go [hunk 1]
```diff
@@ -46,6 +46,7 @@ type ImportStatus struct {
 	PathWithNamespace string     `json:"path_with_namespace"`
 	CreateAt          *time.Time `json:"create_at"`
 	ImportStatus      string     `json:"import_status"`
+	ImportError       string     `json:"import_error"`
```
Comment (svanharmelen) [inline project_import_export.go:49]: Should we also add `CorrelationID` and `FailedRelations`? As I notice that are also mentioned in the docs...

**Review by svanharmelen — CHANGES_REQUESTED** [review]

File: project_import_export.go (hunk 1 above)
Comment (zbindenren) [reply project_import_export.go:49]: Yes that could make sense. I just added the minimum to get the reason when the import fails.

**Review by svanharmelen — APPROVED** [review]
Added missing fields, should be good now. Thanks @zbindenren...

**Pull request merged**